src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+.body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.clock {
+  position: relative;
+  width: 800px;
+  height: 800px;
+  border: 5px dotted black ;
+  border-radius: 50%;
```

Comment:
you can create a `mixin` for creating a circle  since you create a circle twice 

```
@mixin circle($size) {
  width: $size;
  height: $size;
  border-radius: 50%;
}
```